integrations/lib/multiwoven/integrations/source/aws_athena/client.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require "aws-sdk-athena"
```

Comment:
Please add this in `integrations/lib/multiwoven/integrations.rb`

Reply:
remove this require "aws-sdk-athena" from client.rb
its already added in integrations.rb